cloudevents/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,109 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
```

Comment:
I like the spec except for the "turbo" moniker and in spite of my affinity to the automaker that has "turbo" prominently in its model designations. I'd prefer the somewhat more boring "compact".

Reply:
+1 for compact (I should have noted this earlier).

Reply:
There are two intents:

- go fast
- be cheap

Being "compact" is the solution.

Reply:
Done